Updates: small file upload size and link post-workshop

sjspielman opened the pull request:
Two items here:

1) Closes #103 
    - 1 GB --> 100 MB as the file upload limit. Yay or nay on the tildes?

2) Closes #145 
    - Link to the post-workshop environments page from workshop resources
    - As mentioned in this issue, we could also purge "6 months" from other markdowns where it appears. Let me know what you think during review!

(edit)
Bonus 3rd issue! Let's go ahead and close #130 here too. Seemed like we were far enough along to just make this happen now, since the final merge into `main` will spell check anyways.

jashapiro commented on workshop/working-with-your-own-data.md:
```diff
@@ -331,9 +331,9 @@ For the folder or file you want to download, right click on it and choose `Downl
 A progress bar on the bottom of the screen will tell you approximately how long it will take to download.
 
-## Upload *small* files (<1 Gb) from your own computer
+## Upload *small* files (~ <100 MB) from your own computer
```
Fancy typography?
```suggestion
## Upload *small* files (≲100 MB) from your own computer
```

jashapiro reviewed (commented):
> * As mentioned in this issue, we could also purge "6 months" from other markdowns where it appears. Let me know what you think during review!

We can probably leave it (and maybe add it here, as long as we say it elsewhere). I don't expect this policy to change, but if it does we can make any such changes then.

As for the rest: A few inline comments and I had a thought that maybe we should reverse the order of the sections: discuss the the small files first, then the big?

sjspielman commented:
I made several changes in `working-with-your-own-data.md`, both in response to review and then I got excitable and did more...

- I updated some links which seemed like they could use a refresh
- I have updated headers to focus on _transferring_ with subsections for uploading & downloading (see screenshot below). I also updated filezilla installation a bit, and added a section for installing packages from github.
- I updated some screenshots - took some new Mac ones that match the current UI, and I resized a bunch that were rendering way too bigly, if you will.
- I changed all the _Template:_ and _Specific example_ to **bold** to make them stand out a bit more amid all the text.
- I ran the spell check script locally since it's not triggered here anymore, and fixed all the speeling erorrs (went with caps for GitLab in dictionary fyi, since that's how we'll use it if ever, I figured?)

Here's how the TOC looks with the updated headers:


<img width="577" alt="Screenshot 2023-09-26 at 11 19 49 AM" src="https://github.com/AlexsLemonade/training-specific-template/assets/4701111/ae97ef14-6755-4ac1-8780-0d2a527f5788">

jashapiro commented on workshop/screenshots/filezilla-permission.png:
How do your screenshots get so big? When I take a screenshot of a dialog like this it ends up less than half this size (and yes, I have a retina screen). I would not expect screenshots should need to be rescaled, normally (html width may need to be set, but the pngs themselves should not need to be adjusted).

sjspielman replied:
Hm... yours are definitely also saved as `png`, yes? I can't seem to find any other setting for this besides changing the file type which will in turn affect size. Mysterious!

jashapiro reviewed (approved):
Looks good overall. A few little comments/suggestions though. And a question about screenshots.

I don't think I should need to see this again.

sjspielman commented on workshop/working-with-your-own-data.md:
```diff
@@ -445,7 +466,16 @@ You should get a similar successful installation message as in the previous sect
 
 Or if it failed to install, it will give you a `non-zero exit status` message.
 
+#### Installing packages from GitHub repositories
```
Oh nice, I didn't know about `pak`! Definitely seems looking into more generally (maybe once it's not labeled "experimental" anymore..)

sjspielman commented on workshop/working-with-your-own-data.md:
````diff
@@ -184,7 +190,7 @@ Your institution, or whomever gave you access to the server, should have given y
 Here's very general examples info about logging into [`ssh`](https://help.liquidweb.com/s/article/Logging-into-Your-Server-via-Secure-Shell-SSH).
 
-*Template*
+**Template:**
 Generally an ssh login will look something like this:
 ```
 ssh username@server
````
agree, removing the tag

sjspielman commented on workshop/working-with-your-own-data.md:
````diff
@@ -133,48 +138,44 @@ wget --user=<USERNAME> --ask-password '<URL>'
-```
-
-Here we will copy this notebook *from* its location in `shared-data/working-with-your-data` *to* our own `training-modules/` folder.
+Here we will copy the `wget` template script from its location in `~/shared-data/working-with-your-data` *to* our own `training-modules/` folder.
````
```suggestion
Here we will copy the `wget` template script *from* its location in `~/shared-data/working-with-your-data` *to* our own `training-modules/` folder.
```

sjspielman commented on workshop/working-with-your-own-data.md:
```diff
@@ -133,20 +138,21 @@ wget --user=<USERNAME> --ask-password '<URL>'
 To understand how this works, we will practice `cp` with some files already in the RStudio Server.
 
-*Template:*
+**Template:**
```
fixed

sjspielman commented on workshop/working-with-your-own-data.md:
```diff
@@ -200,68 +206,131 @@ If you are unsure of the file path of the data you are looking for, we recommend
-This will `r`ecursively copy all the files in the folder you reference:
+This will `r`ecursively copy all the files in the folder you reference, as in:
 
-*Template:*
```
woops, restored!
